src/main/java/duke/ListCommand.java

```diff
@@ -0,0 +1,34 @@
+package main.java.duke;
+import java.util.ArrayList;
```

Comment:
Separation of package statements as per [guidelines](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements)

```suggestion
package main.java.duke;

import java.util.ArrayList;
```